docs/use-service.md

```diff
@@ -14,10 +14,20 @@ You can request tiles using Mapzen's global CDN:
 
   `https://tile.mapzen.com/mapzen/terrain/v1/terrarium/{z}/{x}/{y}.png`
 
+Supported tile size variants:
+
+  `https://tile.mapzen.com/mapzen/terrain/v1/{256,512,260,516}/terrarium/{z}/{x}/{y}.png`
+
+The 260 and 516 variants contain 2 pixels of buffering on each edge.
+
 ##### Normal
 
   `https://tile.mapzen.com/mapzen/terrain/v1/normal/{z}/{x}/{y}.png`
 
+Supported tile size variants:
```

Comment:
We should indicate these are _optional_ and the default when not specified is 256.